docs/resource-manager-generate-new-resource.md

```diff
@@ -66,10 +70,11 @@ HERE
 The example above consists of the following properties:
 
 * Labels:
-  * `Compute` - (Required) - This is the name of the Service defined in Pandora, it is identical to the directory name containing the Pandora API definitions residing in [ `./api-definitions`](https://github.com/hashicorp/pandora/blob/main/api-definitions)
-  * `2021-11-01` - (Required) - This is the Version of the Service
-  * `VirtualMachines`, `VirtualMachineScaleSets` - (Required) - This is the name of the Resource for which there are Pandora API definitions
-  * `virtual_machine`, `virtual_machine_scale_set` - (Required) - This is the name that the resource should have in the AzureRM provider with the `azurerm` prefix omitted. The prefix will be added automatically during generation
+  * `ContainerService` - (Required) - This is the name of the Service defined in Pandora, it is identical to the directory name containing the Pandora API definitions residing in [ `./api-definitions`](https://github.com/hashicorp/pandora/blob/main/api-definitions)
+  * `2022-09-02-preview`, `2023-03-02-preview` - (Required) - These are the Versions from which the resources are generated from
+  * `Fleets`, `TrustedAccess` - (Required) - This is the name of the Resource for which there are Pandora API definitions
```

Comment:
I realise this was present previously, but reading through here `Resource` is ambiguous in this context - since the rest of the codebase is using `API Resource` to refer to `/{Service}/{APIVersion}/{APIResource}`, we should use that terminology here too:

```suggestion
  * `Fleets`, `TrustedAccess` - (Required) - This is the name of the API Resource(s) within the Service / API Version where the Resource(s) are located.
```